app/femr/data/daos/system/PatientRepository.java

```diff
@@ -246,23 +246,32 @@ public IPatient retrievePatientById(Integer id) {
             Query<? extends IRankedPatientMatch> query = QueryProvider.getRankedPatientMatchQuery();
 
             DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
-            String ageString = dateFormat.format(new Date(age));
+            String ageString = "";
+            if(age != null) {
+                ageString = dateFormat.format(new Date(age));
+            }
+
 
             String sql
-                    = "select id, user_id, first_name, last_name, phone_number, age, sex, address, city, isDeleted, deleted_by_user_id, reason_deleted, ("+
-                    "case when phone_number = " + phone + " then 40 else 0 end + " +
-                    "case when last_name = \"" + lastName +"\" then 15 else 0 end + " +
+                    = "select id, user_id, first_name, last_name, phone_number, age, sex, address, city, isDeleted, deleted_by_user_id, reason_deleted, (";
+            if(phone != null && !phone.equals("")) {
+                sql += "case when phone_number = " + phone + " then 40 else 0 end + ";
```

Comment:
Is there a way to make the null phone check in raw SQL so it doesnt break up the statement like it does?